frontend/src/components/Logo.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+import { Link as RouterLink } from 'react-router-dom'
+import {
+    createStyles,
+    makeStyles,
+    Link,
+  } from '@material-ui/core'
+
+interface Props {
+  theme: string
+}
+
+const stylesInUse = makeStyles((theme) =>
```

Comment:
Parameter never used or am I missing something?

Reply:
I can't see any difference in the application when it's removed

Reply:
The `theme` parameter is only needed if we have to directly access the theme given to `<ThemeProvider>` in App. For example in RegisterForm it's used to get the colors for error and success messages. The theme parameter is also unnecessary in Header and BranchSelector components and can be removed.

Reply:
parameter now removed